packages/commonwealth/client/scripts/views/pages/user_dashboard/CommunityPreviewCard.tsx

```diff
@@ -45,7 +44,7 @@ const CommunityPreviewCard = ({ community }: CommunityPreviewCardProps) => {
         {community.description}
       </CWText>
       {/* if no recently active threads, hide this module altogether */}
-      {!!monthlyThreadCount && (
+      {!monthlyThreadCount && (
```

Comment:
Confused about this logical change -- seems like we've gone to ONLY showing this module when no recently active threads? Maybe comment needs to be fixed?

Reply:
This was a mistake, great catch here.